modules/apps/commerce/commerce-service/src/main/java/com/liferay/commerce/internal/upgrade/v8_9_3/CommerceOrderUpgradeProcess.java

```diff
@@ -0,0 +1,97 @@
+/**
+ * Copyright (c) 2000-present Liferay, Inc. All rights reserved.
+ *
+ * This library is free software; you can redistribute it and/or modify it under
+ * the terms of the GNU Lesser General Public License as published by the Free
+ * Software Foundation; either version 2.1 of the License, or (at your option)
+ * any later version.
+ *
+ * This library is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS
+ * FOR A PARTICULAR PURPOSE. See the GNU Lesser General Public License for more
+ * details.
+ */
+
+package com.liferay.commerce.internal.upgrade.v8_9_3;
+
+import com.liferay.commerce.constants.CommerceOrderConstants;
+import com.liferay.commerce.model.CommerceOrder;
+import com.liferay.commerce.order.status.CommerceOrderStatus;
+import com.liferay.commerce.order.status.CommerceOrderStatusRegistry;
+import com.liferay.commerce.service.CommerceOrderLocalService;
+import com.liferay.portal.kernel.upgrade.UpgradeProcess;
+
+import java.sql.PreparedStatement;
+import java.sql.ResultSet;
+
+/**
+ * @author Brian I. Kim
+ */
+public class CommerceOrderUpgradeProcess extends UpgradeProcess {
+
+	public CommerceOrderUpgradeProcess(
+		CommerceOrderLocalService commerceOrderLocalService,
+		CommerceOrderStatusRegistry commerceOrderStatusRegistry) {
+
+		_commerceOrderLocalService = commerceOrderLocalService;
+		_commerceOrderStatusRegistry = commerceOrderStatusRegistry;
+	}
+
+	@Override
+	protected void doUpgrade() throws Exception {
+		_updateCommerceOrder();
+	}
+
+	private void _updateCommerceOrder() throws Exception {
+		try (PreparedStatement preparedStatement = connection.prepareStatement(
+				"select commerceOrderId, orderStatus from " +
+				"CommerceOrder where orderStatus = 14 or orderStatus = 15 ");
```

Comment:
why don't you use constants here instead of numbers?

Reply:
Hey @andrea-ale-sbarra , you are right. I forgot to change it to constants for this one. I will address it right away.